frontend/src/components/user-details/components/user-profile/components/user-profile-form/helpers/get-gender-options-helper/get-gender-options.helper.ts

```diff
@@ -0,0 +1,14 @@
+import { UserGender } from 'common/enums/enums';
+import { SelectorOptions } from 'common/types/types';
+
+const getGenderOptions = (): SelectorOptions[] => {
+  return Object.values(UserGender).map(
+    (it) =>
+      <SelectorOptions>{
+        name: it,
+        value: it,
+      },
```

Comment:
You can provide resulting type for mapped items in `.map` generic
```suggestion
  return Object.values(UserGender).map<SelectorOptions>(
    (it) => ({
        name: it,
        value: it,
      }),
```